api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/BaseHandler.java

```diff
@@ -51,6 +56,17 @@ public Response baseJSON(@Context ServletContext context) {
         return Response.ok(hawkularMetrics).build();
     }
 
+    @GET
+    @Produces({APPLICATION_XHTML_XML, TEXT_HTML})
+    @ApiOperation(value = "Returns some basic information about the Hawkular Metrics service.",
+                  response = String.class, responseContainer = "Map")
+    public void baseHTML(@Context ServletContext context) throws Exception {
+
+        HttpServletRequest request = ResteasyProviderFactory.getContextData(HttpServletRequest.class);
```

Comment:
I don't believe that will work. '/' and all its descendants are owned by the BaseHander, but we use the servlet-mapping to over ride it for anything under '/static/*'.

If we want to use just the servlet-mapping, then we will also need to create individual entries for all other types of static files (*.css, *.js, *.png, etc). 

I think we would still need to create an entry in BaseHandler anyways to forward '/' to '/index.html' only when the request type is html instead of json.